tests/ListProjectsUsersTest.php

```diff
@@ -0,0 +1,115 @@
+                $this->client->removeUserFromMaintainer($this->testMaintainerId, $member['id']);
+            }
+        }
+
+        $this->organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
+        ]);
+
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => 'spam+spam@keboola.com']);
+        $this->client->removeUserFromOrganization($this->organization['id'], $this->superAdmin['id']);
+    }
+
+    public function testSuperAdminCannotGetList()
+    {
+        $this->expectException(ClientException::class);
+        $this->expectExceptionMessage(self::LIST_ORGANIZATION_PROJECTS_USERS_EXCEPTION_MESSAGE);
+        $this->expectExceptionCode(403);
+
+        $this->client->listOrganizationProjectsUsers($this->organization['id']);
+    }
+
+    public function testMaintainerCannotGetList()
+    {
+        $this->client->addUserToMaintainer($this->testMaintainerId, [
+            'email' => $this->normalUser['email']
+        ]);
+
+        $this->expectException(ClientException::class);
+        $this->expectExceptionMessage(self::LIST_ORGANIZATION_PROJECTS_USERS_EXCEPTION_MESSAGE);
+        $this->expectExceptionCode(403);
+
+        $this->normalUserClient->listOrganizationProjectsUsers($this->organization['id']);
+    }
+
+    public function testOrganizationUserCanGetList()
+    {
+        $this->client->addUserToOrganization($this->organization['id'], [
+            "email" => $this->normalUser['email']
+        ]);
+
+        $testProject = $this->normalUserClient->createProject($this->organization['id'], [
+            'name' => 'Test Project',
+        ]);
+
+        $this->normalUserClient->addUserToProject($testProject['id'], [
+            "email" => 'spam+spam@keboola.com',
+        ]);
+
+        $this->normalUserClient->addUserToProject($testProject['id'], [
+            "email" => $this->superAdmin['email']
+        ]);
+
+        $this->normalUserClient->removeUserFromProject($testProject['id'], $this->normalUser['id']);
+
+        $results = $this->normalUserClient->listOrganizationProjectsUsers($this->organization['id']);
+
+        $this->assertCount(2, $results);
+        foreach ($results as $result) {
+            $this->assertArrayHasKey('id', $result);
+            $this->assertArrayHasKey('name', $result);
+            $this->assertArrayHasKey('email', $result);
+        }
+    }
+
+    public function testUserOneOfProjectInOrg()
+    {
+        $this->client->addUserToOrganization($this->organization['id'], [
+            "email" => $this->normalUser['email']
+        ]);
+
+        $this->normalUserClient->createProject($this->organization['id'], [
+            'name' => 'Test Project',
+        ]);
+
+        $this->normalUserClient->removeUserFromOrganization($this->organization['id'], $this->normalUser['id']);
+
+        $this->expectException(ClientException::class);
+//        @todo You don't have access to the organization
```

Comment:
Tu dostavam tiez inu hlasku bud robim nieco blbo alebo to bude ako v tom dalsom teste

Reply:
Tak som oddelal to todo ma to byt tak ako je to v tom dalsom teste.